v2/cmd/zlint/main.go

```diff
@@ -51,14 +55,16 @@ var ( // flags
 func init() {
 	flag.BoolVar(&listLintsJSON, "list-lints-json", false, "Print lints in JSON format, one per line")
 	flag.BoolVar(&listLintSources, "list-lints-source", false, "Print list of lint sources, one per line")
+	flag.BoolVar(&summary, "summary", false, "Prints a short human-readable summary report")
+	flag.BoolVar(&longsummary, "longsummary", false, "Prints a human-readable summary report with details")
```

Comment:
small nit: I think naming the flag `longSummary` instead of `-longsummary` will better match existing options (`-nameFilter`, `-includeSources`, etc).

Reply:
changed in 832bc81